tests/test_download.py

```diff
@@ -77,12 +74,22 @@
 REPETITIVE_DOC = {'title': 'Yadda', 'text': 'yadda yadda', 'metadata': []}
 
 
+def doc_paring_callback(request, context):
+    fields = json.loads(request.qs['fields'][0])
```

Comment:
I assume this `request.qs()` business is a "get this item out of the query string" method from `requests_mock`?

Reply:
Right.  Do you think it needs a comment to that effect, or is it apparent enough?

Reply:
It couldn't hurt, since it's a bit opaque and I had to think about it for a moment to figure out what it was doing.